internal/core/command/put_test.go

```diff
@@ -73,3 +80,77 @@ func TestNewPutCommandBody(t *testing.T) {
 		t.Error("Failed to verify the request body contents")
 	}
 }
+
+func TestNewPutCommandContentType(t *testing.T) {
+	tests := []struct {
+		name            string
+		originalHeaders map[string]string
+		expectedHeaders map[string]string
+		req             *http.Request
+	}{
+		{
+			"cbor content type header propagated",
+			map[string]string{clients.ContentType: clients.ContentTypeCBOR},
+			map[string]string{clients.ContentType: clients.ContentTypeCBOR},
+			newRequestWithHeaders(map[string]string{clients.ContentType: clients.ContentTypeCBOR}, http.MethodPut),
```

Comment:
I forgot to mention this in the previous round of iteration, but since we are going to have another round of development can we add names to each of these parameters. For example:
```go
name: "cbor content type header propagated",
originalHeaders: map[string]string{clients.ContentType: clients.ContentTypeCBOR},
expectedHeaders: map[string]string{clients.ContentType: clients.ContentTypeCBOR},
req: newRequestWithHeaders(map[string]string{clients.ContentType: clients.ContentTypeCBOR}, http.MethodPut),
```